zerver/tests/test_auth_backends.py

```diff
@@ -2083,7 +2082,7 @@ def test_login_failure_due_to_empty_subdomain(self) -> None:
                 result = self.client_post('http://testserver:9080/accounts/login/sso/',
                                           REMOTE_USER=email)
                 self.assertEqual(result.status_code, 200)
-                self.assertIs(get_session_dict_user(self.client.session), None)
+                self.assert_logged_in_user_id()
```

Comment:
Let's write this as `assert_logged_in_user_id(None)`; I think that's a lot more readable, and the whole point of this change is to make it more obvious what's going on.